cleaned_data/modsbyPID/rftaart:11.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mods xmlns="http://www.loc.gov/mods/v3"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xs="http://www.w3.org/2001/XMLSchema"
+      xsi:schemaLocation="http://www.loc.gov/mods/v3 http://www.loc.gov/standards/mods/v3/mods-3-5.xsd">
+   <identifier type="local">RFTA_Braddock_011</identifier>
+   <identifier type="pid">rftaart:11</identifier>
+   <titleInfo>
+      <title>Escaping the fire</title>
+   </titleInfo>
+   <originInfo>
+      <dateCreated>2021</dateCreated>
+      <dateCreated encoding="edtf" keyDate="yes">2021</dateCreated>
+   </originInfo>
+   <abstract>Seemona and Daniel Whaley kept watching the smoke, thinking the fire was still far away. Their cat hovered near the heating vent because smoke was coming in through the vents. Finally, they could see that one of the lodges below them was on fire. Someone called and told them to leave now. They had a hard time escaping, having waited almost too long to leave. A fallen tree that was on fire beside the road blocked the way out. They could see one of their friends’ houses in flames.</abstract>
+   <name authority="naf"
+         valueURI="http://id.loc.gov/authorities/names/n2002022941">
+      <namePart>Braddock, Paige</namePart>
+      <role>
+         <roleTerm authority="marcrelator"
+                   valueURI="http://id.loc.gov/vocabulary/relators/art">Artist</roleTerm>
+      </role>
+   </name>
+   <subject>
+      <name>
+         <namePart>Whaley, Seemona</namePart>
+      </name>
+   </subject>
+   <subject>
+      <name>
+         <namePart>Whaley, Daniel</namePart>
+      </name>
+   </subject>
+   <physicalDescription>
+      <form authority="aat" valueURI="http://vocab.getty.edu/aat/300033973">drawings (visual works)</form>
+   </physicalDescription>
+   <subject authority="lcsh"
+            valueURI="http://id.loc.gov/authorities/subjects/sh85146723">
+      <topic>Wildfires</topic>
+   </subject>
+   <subject authority="lcsh"
```

Comment:
@logantillman - the authority here ("lcsh") doesn't match what is present in the valueURI ("tgm"). Other than for the first subject column in the export template, can you change the hard coded value to "tgm"?